region/brinstar/pink/Big Pink.json

```
@@ -3155,6 +3178,53 @@
         "Farm the Zeb pipe just above the 'main' left door. Shinecharge on the ledge below, then hop back up and get interrupted by a Zeb."
       ]
     },
+    {
+      "link": [9, 13],
+      "name": "Direct G-Mode Morph Spark Interrupt with Crystal Flash Clip",
+      "entranceCondition": {
+        "comeInWithGMode": {
+          "mode": "direct",
+          "morphed": true
+        }
+      },
+      "requires": [
+        "canComplexGMode",
+        {"or": [
+          "h_artificialMorphIBJ",
+          "Morph",
+          {"and": [
+            "h_artificialMorphSpringBall",
+            {"or": [
+              "HiJump",
+              {"and": [
+                "h_artificialMorphSpringBallBombJump",
+                "h_additionalBomb",
+                "h_additionalBomb",
+                "h_additionalBomb"
+              ]}
+            ]}
+          ]}
+        ]},
+        "h_artificialMorphCrystalFlash",
```

Missing the charge clip logic:
```suggestion
        "h_artificialMorphCrystalFlash",
        {"or": [
          "Morph",
          {"and": [
            "canInsaneJump",
            "canTwoTileSqueeze",
            "canPartialFloorClip",
            "canCeilingClip"
          ]}
        ]},
```